src/commands/deploy.ts

```diff
@@ -84,6 +84,12 @@ export default class Deploy extends Command {
       hidden: true,
       exclusive: ['account', 'environment', 'auto_approve', 'lock', 'force_unlock', 'refresh'],
     }),
+    parameter: flags.string({
+      description: 'Component parameters',
+      exclusive: ['local', 'compose_file'],
```

Comment:
I think we can remove the exclusive aspect. Should still apply to local deploys.